src/components/ViewMoreListBtn/ViewMoreListBtn.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Box, Button } from "@mui/material";
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+import { useStyles } from "./styles";
+
+export default function ViewMoreListBtn({ onViewMore }: { onViewMore: () => void }) {
+  const styles = useStyles();
+  return (
+    <Box
+      className={styles.btnWrapper}
+      onClick={() => {
+        onViewMore();
+      }}
+    >
+      <Button variant="text" color="primary" className={styles.btn}>
```

Comment:
Where is the `onClick` event on the button? Why are you using the onClick on Box instead of the button?